src/FondOfSpryker/Yves/CrossEngage/Model/UriLanguageKey/UriLanguageKeyPluginUsStorePlugin.php

```diff
@@ -13,8 +13,9 @@ class UriLanguageKeyPluginUsStorePlugin implements UriLanguageKeyPluginInterface
      */
     public function getLanguageKey(Store $store): string
     {
-        if (\strpos($store->getStoreName(), '_US') !== false) {
-            return 'us';
+        $language = array_search($store->getCurrentLocale(), $store->getLocales());
+        if ($language !== false) {
+            return $language;
         }
 
         return '';
```

Comment:
In "Newsletter/Model/Resolver/LanguagePrefixResolver::resolve" gibst du anstatt einen leeren String Store::getCurrentLanguage() zurück.

Reply:
I know. Wollte hier deine Logik nicht komplett ändern. Denke du hast dir schon was dabei gedacht einen leeren String zurückzugeben. Sollte aber eh nie der Fall sein, dass hier keine Sprache gefunden wird.